VCFoodDeliveryApp/Models/Cart.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace VCFoodDeliveryApp.Models
+{
+    public class Cart
+    {
+        [StringLength(450, ErrorMessage = "Too many users created")]
+        public string Id { get; set; }
+
+
+        public string ApplicationUserId { get; set; }
+
+
+        public DateTime DeliveryTime { get; set; }
+
+
+        [DataType(DataType.Currency)]
+        public double OrderTotal { get; set; }
```

Comment:
Money should always be stored as `decimal` to avoid floating point rounding quirks.

Reply:
Changed to decimal. That makes sense. Not sure why I thought I should use a double